app/models/sample_attribute.rb

```diff
@@ -53,6 +53,10 @@ def short_pid
     URI.parse(pid).fragment || pid.gsub(/.*\//,'') || pid
   end
 
+  def linked_custom_metadata_type
+    nil
+  end
+
```

Comment:
I'm not sure why this is needed? Looks a bit fishy

Reply:
It's the equivalent of https://github.com/seek4science/seek/blob/9ef544c92b9334156287c353cd0ef9cd1f475c96/app/models/custom_metadata_attribute.rb#L13-L15